math/utils_test.go

```diff
@@ -66,3 +66,72 @@ func TestStdDev(t *testing.T) {
 		})
 	}
 }
+
+func TestPhiSimple(t *testing.T) {
```

Comment:
should we have a TestGradientSimple?

Reply:
Right below we have one equivalent, just dif naming (without the "simple")